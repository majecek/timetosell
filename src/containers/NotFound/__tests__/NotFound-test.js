import React from 'react'
import renderer from 'react-test-renderer'
import NotFound from '../NotFound'

describe('NotFound', () => {
  const result = renderer.create(<NotFound />).toJSON()

  it('should render full html structure correctly', () => {
    expect(result).toMatchSnapshot()
  })
})