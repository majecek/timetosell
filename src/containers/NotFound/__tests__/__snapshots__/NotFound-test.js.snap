exports[`NotFound should render full html structure correctly 1`] = `
<div
  className="container">
  <h1>
    Doh! 404!
  </h1>
  <p>
    These are 
    <em>
      not
    </em>
     the droids you are looking for!
  </p>
</div>
`;
